testutil/utils.go

```diff
@@ -0,0 +1,36 @@
+package testutil
+
+import (
+	"bytes"
+	"context"
+	"encoding/hex"
+
+	"github.com/celestiaorg/celestia-app/pkg/appconsts"
+	"github.com/celestiaorg/nmt/namespace"
+	"github.com/cosmos/cosmos-sdk/client"
+	tmrand "github.com/tendermint/tendermint/libs/rand"
+	rpctypes "github.com/tendermint/tendermint/rpc/core/types"
+)
+
+func RandomValidNamespace() namespace.ID {
+	for {
+		s := tmrand.Bytes(8)
+		if bytes.Compare(s, appconsts.MaxReservedNamespace) > 0 {
```

Comment:
[Non Blocking][Question]
This one is used to create a PFD. Shouldn't we also check if the namespace is not the ns of evidence or?

Reply:
`MaxReservedNamespace` is greater than the  namespace used for transactions, intermediate state roots, or evidence so this conditional will evaluate to `false` if `s` matches any of those namespaces.

In other words, the returned `s` should only be valid message namespaces, `TailPaddingNamespaceID` or `ParitySharesNamespaceID`

https://github.com/celestiaorg/celestia-app/blob/main/pkg/appconsts/appconsts.go#L90-L96

Reply:
Sorry yes, I meant tail padding or parity. Is that alright?

Reply:
nice catch! 6f50ff6dfbf028c7170d4b2f4eef962d16f3b732